sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseCreateForm.java

```diff
@@ -550,6 +551,7 @@ private void setNoneFacility() {
 		FacilityReferenceDto noFacilityRef = FacadeProvider.getFacilityFacade().getByUuid(FacilityDto.NONE_FACILITY_UUID).toReference();
 		facilityCombo.addItem(noFacilityRef);
 		facilityCombo.setValue(noFacilityRef);
+		facilityType.setRequired(false);
```

Comment:
why is this needed?

Reply:
Without this line, this field will be set as required for validation. It happens only from code change. And in this case, in Validation Form, even if Place of Stay is set to `Home`, app will ask us to introduce the Facility Type(as Required).